css/styles.css

```diff
@@ -116,3 +116,259 @@
+  align-items: center;
+  color: #1c1a19;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  background: #fff;
+}
+
+.recent-works {
+  margin: 120px 24px 104px 24px;
+  border-bottom: 1px solid #212121;
+  padding-bottom: 30px;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: auto;
+  grid-gap: 24px;
+  background-color: #fff;
+  padding-bottom: 120px;
+}
+
+.grid-item {
+  padding: 20px;
+}
+
+.placeholder-img {
+  width: 100%;
+  height: auto;
+}
+
+.post-title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  margin: 12px 0;
+}
+
+.post-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.buttons {
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+}
+
+.buttons li {
+  list-style: none;
+  text-align: center;
+  background-color: #fff;
+  margin-right: 12px;
+  padding: 10px 12px 10px 12px;
+  border: 1px solid #8993a4;
+  cursor: pointer;
+}
+
+.buttons li a {
+  text-decoration: none;
+  color: #091e42;
+}
+
+.see-button li {
+  margin-top: 10px;
+  background: #ff6b00;
+  border: 0;
+  font-weight: 700;
+  font-size: 17px;
+}
+
+.see-button li a {
+  color: #fff;
+}
+
+.art-printing {
+  background-image:
+    linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%),
+    url("../img/works-img/art-printing.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.art-title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 60px 0 0 0;
+}
+
+.art-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.button-transparent li {
+  background: rgba(255, 255, 255, 0.24);
+  color: #fff;
+}
+
+.button-transparent li a {
+  color: #fff;
+}
+
+.see-button-2 {
+  background: #ff6b00;
+  margin-top: 22px;
+}
+
+.see-button-2 li {
+  margin-left: 114px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.about {
+  background: #1c1a19;
+}
+
+.about-me {
+  display: flex;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6f00;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 24px 0;
+}
+
+.about-desc {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 44px 0;
+}
+
+.about-img {
```

Comment:
- Kindly make sure there are no extra margins in the design. As you can see there is an extra white margin on the right. Which display some content to the right and some to left. Make sure to fix it and align it correctly with correct margins.

Your design:
![image](https://user-images.githubusercontent.com/74744787/158995857-6855263d-086e-4638-8f91-2502e634f61c.png)

![image](https://user-images.githubusercontent.com/74744787/158996069-285a578b-f9cf-4581-be45-5dd6a22218e8.png)

- one way to fix it is to add this code to the body and HTML selector :
width: 100%;
    height: 100%;
    margin: 0;
    padding: 0;
    overflow-x: hidden;

Reply:
Thanks for the suggestion. Also, I also found out that, setting `margin` instead of `padding` in `grid-item` helped with this.